### ISSUE TITLE
LC0072 with TryFunction

### ISSUE THREAD

**Issue by fvet** [StefanMaron/BusinessCentral.LinterCop#1205]
What's your take on documenting TryFunctions ?

```
/// <summary>
    /// Checks if a column value is a valid GUID.
    /// </summary>
    /// <param name="ColumnName">The name of the column to check.</param>
    /// <returns>True if the value is a valid GUID, false otherwise.</returns>
    [TryFunction]
    procedure IsValueGuid(ColumnName: Text)
    var
        TempValue: Guid;
        jv: JsonValue;
    begin
        jv := GetValue(ColumnName, false);
        Evaluate(TempValue, jv.AsText());
    end;
```

LC0072 is triggered based on the <returns> documentation part

<img width="834" height="156" alt="Image" src="https://github.com/user-attachments/assets/b76ce130-b28d-41c4-8d3b-4f162b4508be" />

Although the compiler implicitly adds  'Boolean' as return value if the TryFunction function attribute is present.

<img width="673" height="291" alt="Image" src="https://github.com/user-attachments/assets/a393af1e-6be0-4139-8733-162e6542fcd9" />

FYI: Without TryFunction function attribute

<img width="573" height="92" alt="Image" src="https://github.com/user-attachments/assets/e9d53ff6-bbc3-47fd-9bd2-20d9c437b26a" />

**Comment by Arthurvdv** [comment]
I believe you're absolutely right and the rule should handle TryFunctions.

**Comment by Arthurvdv** [comment]
Moved to ALCops
